bears/generate_package.py

```diff
@@ -108,14 +152,20 @@ def main():
     else:  # pragma: no cover
         bear_version = repr(bear_version) + '.' + str(int(time.time()))
 
-    for bear_file_name in sorted(set(glob('bears/**/*Bear.py'))):
+    for bear_file_name in get_bear_glob(args.conda):
         bear_object = next(iimport_objects(
             bear_file_name, attributes='kind', local=True),
             None)
         if bear_object:
             bear_name, _ = os.path.splitext(os.path.basename(bear_file_name))
-            create_file_structure_for_packages(
-                os.path.join('bears', 'upload'), bear_file_name, bear_name)
+            (create_file_structure_for_packages(
```

Comment:
This construct is very confusing ...
Doing a verbose: 

```
if abc:
    a = 1
else:
    a = 2
```
would be much simpler !